github-electron/github-electron.d.ts

```diff
@@ -1718,6 +1718,7 @@ declare module GitHubElectron {
 		powerMonitor: NodeJS.EventEmitter;
 		protocol: GitHubElectron.Protocol;
 		Tray: typeof GitHubElectron.Tray;
+		hideInternalModules(): any;
```

Comment:
I think return type should be `void`.

https://github.com/atom/electron/blob/3a5160f79968998ef214eb1d81c00993b0737637/atom/common/api/lib/exports/electron.coffee#L2